linode/manager
# M3-5813: Types and related adjustments for PostgreSQL and MongoDB

## Description
- Add new types and tweak existing types to accommodate MongoDB and PostgreSQL
- Tweak mocks, factories, and schema accordingly

## How to test
~~Pointing to prod, test the creation of MongoDB, PostgreSQL, and MySQL databases. Ensure that the pricing is correct and that creation proceeds smoothly and as expected. Double-check the console for errors.~~ Cross-check the types and schema with the spec docs sent to the channel last week.

A few points:
- Mongo and Postgres pricing isn't available in dev right now
- Creating a MySQL and Postgres DB is possible in dev. I'm looking into the Mongo issue right now
